components/desktop/e/enlightenment/Makefile

```diff
@@ -45,6 +45,7 @@ CONFIGURE_BINDIR.64=$(USRBINDIR)
 CONFIGURE_OPTIONS+=	--sysconfdir=/etc
 CONFIGURE_OPTIONS+=	--enable-shared
 CONFIGURE_OPTIONS+=	--disable-static
+CONFIGURE_OPTIONS+= --disable-cpufreq
```

Comment:
This module is a no-op on illumos. It currently only supports Linux sysfs and *BSD sysctl.

Reply:
OK